packages/node/test/manual/express-scope-separation/start.js

```diff
@@ -32,22 +32,13 @@ Sentry.init({
   dsn: 'http://test@example.com/1337',
   transport: DummyTransport,
   beforeSend(event) {
-    if (event.message === 'Error: foo') {
-      assertTags(event.tags, {
-        global: 'wat',
-        foo: 'wat',
-      });
-    } else if (event.message === 'Error: bar') {
-      assertTags(event.tags, {
-        global: 'wat',
-        bar: 'wat',
-      });
-    } else if (event.message === 'Error: baz') {
-      assertTags(event.tags, {
-        global: 'wat',
-        baz: 'wat',
-      });
-    }
+    const errorValue = event.exception?.values?.[0].value;
```

Comment:
We can't use optional chaining here as we have to support old node versions in tests.

Reply:
yes 🤦 updated the PR to use transactions for simplicity.